src/store/categories/reduser.js

```diff
@@ -10,6 +15,15 @@ export const categories = (state = store.categories, action) => {
   return state;
 };
 
+export const category = (state = store.category, action) => {
+  switch (action.type) {
+    case SET_CATEGORY: return action.data;
+    case REMOVE_CATEGORIES: return [];
```

Comment:
Use `store.category` as a cleaned state. Also why do you use for category `REMOVE_CATEGORIES` action?